aptos-move/framework/PROVER-GUIDE.md

```diff
@@ -0,0 +1,38 @@
+This guide collects links to the prover documents and gives some hints to troubleshoot issues when using the prover. 
```

Comment:
... using the prover for specifying the Aptos frameworks.

So we avoid someone mistakes this as general docs.